### ISSUE TITLE
Fabric API key not valid. Your Fabric run script build phase should contain your API key:

### ISSUE THREAD

**Issue by devzhr** [cuappdev/ithaca-transit-ios#188]
```
Print: Entry, ":fabric-api-key", Does Not Exist
Print: Entry, ":fabric-build-secret", Does Not Exist
2018-12-08 17:31:22.711 uploadDSYM[9955:219625] Fabric.framework/run 1.8.0 (212)
error: Fabric: Configuration Issue

Fabric API key not valid. Your Fabric run script build phase should contain your API key:
./Fabric.framework/run INSERT_YOUR_API_KEY INSERT_YOUR_BUILD_SECRET
```

i add Keys.plist and fabric-api-key and fabric-build-secret but some problem

**Comment by mattbarker016** [comment]
Is the file located in `/TCAT/Supporting Files` ?

<img width="338" alt="image" src="https://user-images.githubusercontent.com/8039928/49692797-2f47bd00-fb31-11e8-83d0-ed389601a273.png">

**Comment by devzhr** [comment]
yes I added like you on the file Keys.plist but problem remains the same

**Comment by mattbarker016** [comment]
@zouhair2015 I'm sorry for not getting back to you! I just tried cloning the repo and following the build instructions. I believe the step that you are stuck on is failing because there's an issue with the Fabric API information you are providing. I tried putting random text as the value for these keys, and got the same error you had. Make sure you have a valid Fabric API Key, and give it another shot.

Let me know if that solves your problem! We'll try to make this process easier for people that want to dive into our codebase too, thank you!
